cl/search/management/commands/sweep_indexer.py

```diff
@@ -0,0 +1,418 @@
+from datetime import datetime
+from typing import Iterable, Literal, Mapping, cast
+
+from django.apps import apps
+from django.conf import settings
+from django.db.models import QuerySet
+
+from cl.audio.models import Audio
+from cl.lib.celery_utils import CeleryThrottle
+from cl.lib.command_utils import VerboseCommand, logger
+from cl.lib.redis_utils import get_redis_interface
+from cl.people_db.models import Person
+from cl.search.documents import (
+    ES_CHILD_ID,
+    AudioDocument,
+    DocketDocument,
+    ESRECAPDocument,
+    OpinionClusterDocument,
+    OpinionDocument,
+    PersonDocument,
+)
+from cl.search.models import SEARCH_TYPES, Opinion, RECAPDocument
+from cl.search.tasks import (
+    index_parent_and_child_docs,
+    index_parent_or_child_docs,
+)
+from cl.search.types import ESDocumentClassType
+
+supported_models = [
+    "audio.Audio",
+    "people_db.Person",
+    "search.OpinionCluster",
+    "search.Opinion",
+    "search.Docket",
+    "search.RECAPDocument",
+]
+r = get_redis_interface("CACHE")
+
+
+def compose_indexer_redis_key() -> str:
+    """Compose the redis key for the sweep indexer
+    :return: A Redis key as a string.
+    """
+    return f"es_sweep_indexer:log"
+
+
+def log_indexer_last_status(
+    model_name: str, document_pk: int, chunk_size: int, log_key: str
```

Comment:
Instead of passing `log_key` as a parameter to this method, could we achieve the same result by using the `compose_indexer_redis_key` helper directly within the method to compute the `key`?

Reply:
Yeah, this is possible. Changed!